runtime/ftplugin/solidity.vim

```diff
@@ -0,0 +1,7 @@
+" Vim filetype plugin file
+" Language: Solidity
+" Previous Maintainer: Cothi (jiungdev@gmail.com) 
+" Original author: tomlion (https://github.com/tomlion/vim-solidity)
+" Last Changed: 2022 Sep 27
+
+setlocal commentstring=//\ %s
```

Comment:
`b:undo_ftplugin` needs to be set. See `:help undo_ftplugin`

Setting `'comments'` as well while you're here would be a good idea.